advanced-usage/index.md

```diff
@@ -60,21 +61,44 @@ On the right side of the header you'll find:
 
 Choose between:
 - A static IP
-- A DHCP server
 - A dynamic IP
+- A DHCP server
 
 {{ easy_image(src="ethernet", width=350) }}
-{{ easy_image(src="ethernet-example", width=250) }}
 
-
-##### Wifi network management
+##### Wifi + Hotspot network management
 {{ service(service="Wifi Manager", port=9000, link="/services/wifi", based=true) }}
 
-- Choose a network to connect to
-- Forget, connect to, or a force a new password for a saved network
+- Choose a wifi network to connect to
+   - The BlueOS web interface is accessible via 
+   [http://blueos-wifi.local](http://blueos-wifi.local) when
```

Comment:
it would be cool if we could ping the address to check if it is actually available as the user browses...
I'm not asking you to do it, though =P

Reply:
haha, as patrick would say, "open an issue" :-P